libvirt/domain.go

```diff
@@ -473,6 +475,14 @@ func setGraphics(d *schema.ResourceData, domainDef *libvirtxml.Domain, arch stri
 				VNC: &libvirtxml.DomainGraphicVNC{},
 			}
 			domainDef.Devices.Graphics[0].VNC.AutoPort = formatBoolYesNo(autoport)
+			if autoport == false {
```

Comment:
a comment of this would be good. e.g would be good to have a comment what `-1` means in case of autoport